src/components/NoClassBox/NoClassMessageBox.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import "./NoClassMessageBox.css";
+import { IoAddCircle } from "react-icons/io5";
+
+export default function NoClassMessageBox(props) {
+  const { message } = props;
```

Comment:
You could use destructuring here to save one line.
```suggestion
export default function NoClassMessageBox({ message } ) {
```